### ISSUE TITLE
boost test application can't be compiled

### ISSUE THREAD

**Issue by brucellino** [SouthAfricaDigitalScience/boost-deploy#1]
From build 10, we can't find the header files when compiling a test application.

in `check-build.sh` : 

```
c++ -I${WORKSPACE}/${NAME}_${REMOTE_VERSION}/include -L${WORKSPACE}/${NAME}_${REMOTE_VERSION}/lib hello-world.cpp
```

Error: 

```
hello-world.cpp:1:35: fatal error: boost/iostreams/tee.hpp: No such file or directory
 #include <boost/iostreams/tee.hpp>
```

There is something wrong with the modulefile for boost.

**Comment by brucellino** [comment]
fixed in e75cad3a1

**Comment by brucellino** [comment]
included in build 46
